### ISSUE TITLE
Standardize the help formatting

### ISSUE THREAD

**Issue by GarthDB** [topcoat/topdoc#41]
@misterbrownlee I noticed you styled the help output a differently. It would be nice to standardize on something.

**Comment by GarthDB** [comment]
@misterbrownlee what do you think of http://docopt.org/

**Comment by misterbrownlee** [comment]
I was trying to follow docopt ... I'm not going to take it personally if you second guess my interpretation?  Or, if you want to discuss whatever specific comment your aren't sure about, we can make the sausage right here in the open 🔨

**Comment by GarthDB** [comment]
I like it a lot. I hadn't seen it before I went looking for a standard.

I'll have to spend some time getting into it.

I did find an issue with the command line arguments. The commander style isn't automatically being interpreted correctly by the config file loader so abbreviated commands aren't overriding the rc file. I'll have to create a new issue for that.

**Comment by misterbrownlee** [comment]
good times.  I do think making things default to false with a flag to switch them on makes the API cleaner.  I think that's the way you went with clobber.  I'll take a closer look at the tests so I can write my own next time

**Comment by GarthDB** [comment]
I fixed the command line arguments bug - I made a [helper function](https://github.com/garthdb/commander-rc-adapter) for commander.js and rc to play together nicely.

I just published it in the latest beta (1.0.4), so it would probably be best to use that to avoid the short name conflict in the future.

**Comment by GarthDB** [comment]
So it looks like docopt has weird stuff with options - it doesn't let you specify optional values or different types.

It wants them in this format:

```
--coefficient=K  The K coefficient [default: 2.95]
--output=FILE    Output file [default: test.txt]
--directory=DIR  Some directory [default: ./]
```
